src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -3234,6 +3234,34 @@ include::./references/DataConnectors.adoc[]
 
 include::./references/ApplicationConnectors.adoc[]
 
+== Integration of ProActive Services in Third-party User Portals
+In this section, we will discuss how ProActive Services can be integrated into third-party user portals. We will cover the following topics:
+
+* Submission of a Workflow in a Dedicated Window: How to use an external window for workflow submission outside our software.
+* Monitoring the Execution of a Specific Job in a Dedicated Window: How to view monitoring data and workflow information after submission.
+
+=== Submission of a workflow in a dedicated window
+We offer an external window that can be accessed outside our software using the following URL format:
+
+----
+/automation-dashboard/#/submit?sessionid={sessionid}&name={workflowName}&bucket={bucketName}
+----
+
```

Comment:
this is unclear as by reading it I understand that all these parameters are mandatory, while they are all optional

Reply:
you should differenciate use cases:

- submit any workflow, lettting the user choose the bucket and workflow he/she wants to submit
- submit any workflow from a specific bucket
- submit a single workflow (bucket and workflow are determined)

you can mention that in all the above scenarios, the sessionid can either be provided in the URL or, if not provided, the user will need to login by entering his/her credentials